mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -454,11 +450,8 @@ def init_roi_field(self, roi_field: QLineEdit):
         if self.stack is None:
             return
 
-        larger = np.greater(self.stack.data[0].shape, (200, 200))
-        if all(larger):
-            return
-        x = min(self.stack.data[0].shape[0], 200)
-        y = min(self.stack.data[0].shape[1], 200)
+        x = min(self.stack.data[0].shape[0], self.stack.data[0].shape[0]) // 2
+        y = min(self.stack.data[0].shape[1], self.stack.data[0].shape[1]) // 2
```

Comment:
Is the `min` doing anything? Could these be
`x = self.stack.data[0].shape[0] // 2`
or just
`x = self.stack.data.shape[1] // 2`

Reply:
probably not anymore. I'll have a look and probably remove, thanks for picking up on this